src/alert-monitoring/getStatus.js

```diff
@@ -0,0 +1,110 @@
+import fetch from "node-fetch"
+import { createRequire } from "module"; // Bring in the ability to create the 'require' method
+const require = createRequire(import.meta.url);
+globalThis.fetch = fetch
+
+function delay(t) {
+    return new Promise(resolve => setTimeout(resolve, t));
+}
+
+const GRAPH_NODE="";
+let indexerQuery = `{ 
+        indexingStatuses  { 
+            subgraph synced health entityCount 
+            chains { 
+                latestBlock { number }
+                network
+            } 
+            fatalError {
+                message
+                block { hash number }
+                handler deterministic
+            }
+        } 
+    }`
+
+export async function getSubgraphData(indexerGraphQLURL) {
+    const data = JSON.stringify({query: indexerQuery});
+    const response = await fetch(
+        indexerGraphQLURL,
+        {
+            method: 'post',
+            body: data,
+            headers: {
+                'Content-Type': 'application/json',
+                'Content-Length': data.length,
+                'User-Agent': 'Node',
+            },
+        }
+    );
+
+    const status = response.status;
+    console.log("Status: " + status);
+
+    if (status === 200) {
+        const queryResult = await response.json();
+        console.log(`${JSON.stringify(queryResult["data"]["indexingStatuses"])}`)
+        return queryResult["data"]["indexingStatuses"]
+    } else {
+        return null
+    }
+}
+
+export function getSubgraphError(allIndexedSubgraphs, archiveNodeLatestBlock, currentNetwork){
+    let errorMessageList = []
+    for (let idx = 0; idx < allIndexedSubgraphs.length; idx++) {
+        let subgraphId = allIndexedSubgraphs[idx]["subgraph"]
+        console.log(`Processing subgraph ${subgraphId} on network ${currentNetwork}`);
+        if (allIndexedSubgraphs[idx]["fatalError"] !== null){
+            return allIndexedSubgraphs[idx]["fatalError"]
```

Comment:
add semicolon in end wherever missing.
allIndexedSubgraphs[idx]["fatalError"] => allIndexedSubgraphs[idx]["fatalError"];